src/index.js

```diff
@@ -13,7 +14,18 @@ ReactDOM.render(
   <React.StrictMode>
     <BrowserRouter>
       <header>
-        <h1>Rotten<br/> Tomatillos</h1>
+        <h1>
```

Comment:
It probably makes more sense to wrap the Link around the H1, rather than wrapping the H1 around the Link